lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.ts

```diff
@@ -44,6 +48,9 @@ import { TagsCreatorMode } from '../../../tag/tags-creator/tags-creator-mode';
 import { TagService } from '../../../tag/services/tag.service';
 import { CategoryService } from '../../../category/services/category.service';
 import { CategoriesManagementMode } from '../../../category/categories-management/categories-management-mode';
+import { MatExpansionPanel } from '@angular/material/expansion';
+import { AllowableOperationsEnum, ContentService } from '@alfresco/adf-content-services';
```

Comment:
incorrect import, this might break with circular dependency during productions builds... this file is already in the content-services, and should use '../../' import

Reply:
I have updated the import path